.github/SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+Thank you for helping us keep our software safe and secure. We value the security community and the important work you do. This document outlines our approach to handling security vulnerabilities.
+
+## Supported Versions
+
+## Reporting a Vulnerability
+
+**Please responsibly disclose any potential security vulnerabilities by mailing the following address:**
+
+infosec@form3.tech
+
+**When reporting a vulnerability, we request that you:**
+
+* **Provide detailed information:** Include descriptions of the vulnerability, the affected software versions, and any steps to reproduce the isse.
+* **Technical details:** Share relevant code examples, screenshots, and any materials to help us understand the problem.
+* **Exercise patience:** Please allow us a reasonable time to investigate and respond to your report making any public disclosures.
+* **Act in good faith:** Avoid any actions that could disrupt services or compromise data for our users.
+
+## Our commitment
+
+* We will acknowledge the receipt of your vulnerability report as soon as possible.
+* We will work diligently to investigate and provide updates on the reported vulnerability in a timely manner.
+* We will collaborate with you to validate and understand the full impact of the vulnerability.
```

Comment:
Suggestions on point 3-5: 
* If you are willing we will collaborate with you to validate and understand the full impact of the vulnerability.
* If appropriate, we will release security patches or updates to address the reported vulnerability, which you will be notified of and in some cases you may be invited to confirm that the solution covers the vulnerability you have raised.
* We may choose to publicly acknowledge your responsible disclosure, if you are agreeable. If you are we would like to unify our release with you so please continue to coordinate with us.